src/script/miniscript.h

```diff
@@ -549,10 +551,10 @@ struct Node {
     Type GetType() const { return typ; }
 
     //! Check whether this node is valid at all.
-    bool IsValid() const { return !(GetType() == ""_mst); }
+    bool IsValid() const { return !(GetType() == ""_mst) && ScriptSize() <= MAX_STANDARD_P2WSH_SCRIPT_SIZE; }
```

Comment:
In commit "Miniscript: conversion from script"

I think this change (as well as the one to `IsValidTopLevel` below) is in the wrong commit.